tests/test_insert_deletes_consolidate.cpp

```diff
@@ -141,16 +193,16 @@ void delete_from_beginning(diskann::Index<T, TagT>& index,
 }
 
 template<typename T>
-void build_incremental_index(
-    const std::string& data_path, const unsigned L, const unsigned R,
-    const float alpha, const unsigned thread_count, size_t points_to_skip,
-    size_t max_points_to_insert, size_t beginning_index_size,
-    size_t points_per_checkpoint, size_t checkpoints_per_snapshot,
-    const std::string& save_path, size_t points_to_delete_from_beginning,
-    size_t start_deletes_after, bool concurrent) {
-  const unsigned C = 500;
-  const bool     saturate_graph = false;
-
+void build_incremental_index(const std::string& data_path, const unsigned L,
+                             const unsigned R, const float alpha,
+                             const unsigned thread_count, size_t points_to_skip,
+                             size_t max_points, float insert_percentage,
+                             float              build_percentage,
+                             const std::string& save_path,
+                             float              delete_percentage,
+                             size_t start_deletes_after, bool concurrent) {
+  const unsigned      C = 500;
```

Comment:
please don't change the origin format if there is no code change here, it make the review inconvenience.

Reply:
Because we want to enter the percentage of deleted and inserted data